inventories/README.md

```diff
@@ -2,30 +2,39 @@
-In addition to this a file to configure the firewall rules for the cluster's network is needed
-
 # Inventories directory
 
-This is the place where you can store your inventories.
-In this folder, different templates are provided:
-- The cluster description inventory, where you will populate the details of your nodes (ip, interface, network details, disks, etc.): seapath_cluster_definition_example.yml
-- The basic cluster inventory for POC: seapath_basic_cluster_definition_example.yml
-- A standalone description inventory: seapath_standalone_definition_example
-- The OpenVSwitch topology inventory, where you will describe the different bridges and ports configuration for OVS: seapath_ovstopology_definition_example.yml
-- The VM inventory, where you will describe all variables related to the virtual machine (number of cores, features, ip, network interface, etc.): seapath_vm_definition_example.yml
-- The firewall rules, where you will describe the packet filtering and the NAT on the system: iptables_rules_example.txt
+On SEAPATH, the definition of our setup must be described inside [Ansible inventories](https://docs.ansible.com/ansible/latest/inventory_guide/intro_inventory.html#passing-multiple-inventory-sources).
+You have to define your own inventories files which match with your setup.
+
+In this directory, you can find some commented examples of inventories for the following case:
+* a demo SEAPATH cluster which can be used to easily test SEAPATH with retails machines (use it as a POC, not in production)
+* a SEAPATH standalone hypervisor setup
+* a complex SEAPATH cluster
+
+It is possible with Ansible to provides [multiple inventories files](https://docs.ansible.com/ansible/latest/inventory_guide/intro_inventory.html#passing-multiple-inventory-sources), and we recommend using this feature for complex setup. In this directory, we have separated the "complex SEAPATH cluster" into three inventories. We have a dedicated inventory file for:
+* the physical machines
+* the cluster internal network (Open vSwitch topology)
+* the virtual machines
+## SEAPATH Ansible inventories examples description
+
+All inventories examples are in YAML format. The example inventories are:
+* `seapath_demo_cluster_definition_example.yml`: demo SEAPATH cluster
+* `seapath_standalone_definition_example`: SEAPATH standalone hypervisor setup
+* The complex SEAPATH cluster inventories are:
+	* `seapath_cluster_definition_example.yml`: physical machines
+	* `seapath_ovstopology_definition_example.yml`: cluster internal network (Open vSwitch topology)
+	* `seapath_vm_definition_example.yml`: virtual machines
 
+The directory also contains the firewall rules `iptables_rules_example.txt` which are referring in `seapath_cluster_definition_example.yml`.
 
-## Optionnal variables
+Except the simple SEAPATH cluster inventory which require minor tweak to be adapted, in all other files we described all the options which do not match a realistic case.
```

Comment:
minors tweaks

>  in all other files we described all the options which do not match a realistic case.

Strange formulation difficult to understand

Reply:
corrected by "The simple SEAPATH cluster inventory require only minor tweak to be adapted. In all other files described, all the options which do not match a realistic case and need to be modified are indicated."